service/telemetry/generated_schema.go

```diff
@@ -0,0 +1,176 @@
+// Code generated by github.com/atombender/go-jsonschema, DO NOT EDIT.
```

Comment:
What command generated this?  Do we need it in the makefile?

Reply:
yup! will add go-jsonschema to the internal tools in a separate PR

Reply:
Added here https://github.com/open-telemetry/opentelemetry-collector/pull/7679